textile/features.textile

```diff
@@ -1532,6 +1539,22 @@ h4. BatchPresenceResult
 ** @(BGR2b)@ @presence@ @PresenceMessage[]@ - an optional array containing all members present on the channel if the request succeeded
 ** @(BGR2c)@ @error@ @ErrorInfo@ - an optional @ErrorInfo@ indicating the reason the presence request failed for the given channel, if unsuccessful
 
+h4. TokenRevocationTarget
```

Comment:
Might `TokenRevocationTargetSpecifier` be more appropriate, given this is the language used in the [REST API documentation](https://ably.com/docs/core-features/authentication#target-specifiers)? Or are we using the words "target" and "target specifier" interchangeably?

Reply:
Good point - I've changed it to `TokenRevocationTargetSpecifier`